### PR TITLE
Fixed bug with bottom buttons moving in order form

### PR DESCRIPTION
**Pull request by jacob-butler**

### CODE REVIEW

**Comment by DuaneShaffer** [comment]
D.r.y.

**Comment by jacob-butler** [comment]
There isn't really a way to make it dryer (at least not using v-if to show the buttons). Each button is unique because you can't separate the on-click from the v-if, and the on-click changes depending on the which element you are on. I did find a typo in the submit button though :)

**Comment by DuaneShaffer** [comment]
Have you tried something along the lines of element += 1 / element -= 1?
Looks like the back button text doesn't change and the next button text could either be next or you could display the name of the v-stepper element.

**Comment by jacob-butler** [comment]
I wasn't aware that you could use +=/-= on the element. I also didn't know that v-if supported && and ||. I was able to refractory it down to 3 buttons. If one were to make the back button change colors it would be possible to make it two buttons but not really worth the effort.

File: website/src/components/Order.vue [hunk 1]
```diff
@@ -90,6 +32,50 @@
...
+          v-on:click="element == 3 ?
+            onSubmitOrder(order) : element = 3"
+          fixed
+        >  
+          <span v-if="element == 2">
+            Review Order
+          </span>
+          <template v-if="element == 3">
+            Submit Order
+          </template>
```
Comment (DuaneShaffer) [inline website/src/components/Order.vue]: Why is one v-if in a span and the other in a template?

File: website/src/components/Order.vue [hunk 2]
```diff
@@ -139,7 +125,6 @@ export default {
         );
     }
   }
-
```
Comment (DuaneShaffer) [inline website/src/components/Order.vue]: Nice catch.

File: website/src/components/Order.vue [hunk 3]
```diff
@@ -90,6 +32,50 @@
       </v-stepper-content>
 
       </v-stepper-items>
+      <v-stepper-items>
```
Comment (DuaneShaffer) [inline website/src/components/Order.vue]: Don't need to be v-stepper-items

**Pull request merged**